### PR TITLE
Refactor Autodeploy

### PR DESCRIPTION
**Pull request by gMonty030**
- Removed Dockerhub autodeploy weebhooks
- Added new step for the autodeploy request after the image is built
- Added auth to autodeploy service, now is more secure the deployment of new versions.

### CODE REVIEW

File: .github/workflows/python.yml [hunk 1]
```diff
@@ -106,3 +106,23 @@ jobs:
       run: bash scripts/deploy_docker.sh ${GITHUB_REF##*/}
       env:
         DOCKERHUB_PROJECT: safe-transaction-service
+  autodeploy:
+    runs-on: ubuntu-latest
+    needs: [test-app, docker-deploy]
```
Comment (Uxio0) [inline .github/workflows/python.yml:111]: Only `docker-deploy` is needed, as it depends on `test-app`

**Review by Uxio0 — CHANGES_REQUESTED** [review]

**Review by Uxio0 — APPROVED** [review]

**Review by Uxio0 — APPROVED** [review]

File: .github/workflows/python.yml (hunk 1 above)
Comment (gMonty030) [reply .github/workflows/python.yml:111]: got it

**Comment by Uxio0** [comment]
recheck

**Comment by gMonty030** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**